doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3863,7 +3948,20 @@ Bootloader
 
 The issues in this section are related to :ref:`app_bootloaders`.
 
-.. rst-class:: v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
+.. rst-class:: v3-1-0
+
+NCSDK-19265: Errors and faults in retrieving the NV counter results in disabled rollback protection
+  The rollback protection in NSIB does not validate counter values if the values cannot be retrieved correctly or if they are larger than the largest supported value.
+  In such a case, the image booted might be one whose values is lower than expected NV counter value.
+
+.. rst-class:: v3-1-0
+
+NCSDK-31918: NSIB active slot function can give invalid results
+  The function for getting the active s0 or s1 slot only checks the :c:struct:`fw_info` structure.
+  The image itself is not validated nor is NSIB queried for which slot is active.
+  This means that if an image is invalidated but has a higher firmware version or if the image is corrupt or even entirely missing (except for the :c:struct:`fw_info` structure), the response to this query will return the wrong image, possibly leading to the wrong update being loaded or impossibility to load a firmware update (depending upon firmware configuration).
+
+.. rst-class:: v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0
 
 NCSDK-31918: Monotonic counter update protection counter limit does not work
```

Comment:
ah I just noticed `NCSDK-31918` is already listed here :|, could you update description with the `NCSDK-19265` contents from above then change `NCSDK-31918` above to `NCSDK-19265` and also put the same version numbers from here to the above bug? Sorry